.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,68 @@
+name: 🐞 Bug
+description: Create a report to help us improve
+title: "<title>"
+labels: ["Bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Version
+      placeholder: |
+        $ kamu version
+        appVersion: 0.150.0
+        buildTimestamp: 2023-12-29T10:41:15.288818966Z
```

Comment:
Maybe also GIT hash:
`gitSha: 0db95273bed61f322259ff6f09906b8755915fbb`

Reply:
I believe the suggestion is for reporter to paste the entire ouput of `kamu version` command.

I suggest to say that directly, e.g. here's an example from `podamn` repo:
![image](https://github.com/kamu-data/kamu-cli/assets/204914/aba6737a-60a0-4f0b-93a4-8831330b9a6f)

Reply:
Fixed via 5232927e7e596bafca6a35caae7754e7400490a2